LongestSubstring_Test.java

```diff
@@ -0,0 +1,23 @@
+package Week1_Longest_Substring_without_repeat;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import junit.framework.Assert;
+
+class LongestSubstring_Test {
+
+	@Test
+	void test() {

```

Comment:
It can be helpful to divide up your tests into different methods that describe what in particular is being tested. That way when there is a failure, you know exactly what failed from the method name instead of trying to figure out what the issue was by finding the broken assert.

Reply:
So, I should create different methods for each assert statement, right